src/containers/org/MemberRolesContainer.js

```diff
@@ -11,7 +11,7 @@ import { RemoveRoleFromMemberModal } from './components';
 import { isRoleAssignedToMember } from './utils';
 
 import { StackGrid } from '../../components';
-import { selectCurrentUserOrgOwner } from '../../core/redux/selectors';
+import { selectOrganizationIsOwner } from '../../core/redux/selectors';
```

Comment:
Hm didn't realize this was a duplicate because this naming doesn't make any sense. It reads as if organizations can be owners

Reply:
Renamed.